services/ux-backend/main.go

```diff
@@ -94,3 +106,22 @@ func main() {
 	log.Fatal(err)
 
 }
+
+func newClient() (client.Client, error) {
```

Comment:
In this file, all private function goes above main, not below. 
Let's stay consistent

Reply:
ack